danger/dangerfile.js

```diff
@@ -93,6 +93,35 @@ if (!includesTestPlan && !editsDocs) {
   markdown('@facebook-github-bot label Needs more information');
 }
 
+// Regex looks for given categories, types, a file/framework/component, and a message - broken into 4 capture groups
+const releaseNotesRegex = /\[(ANDROID|CLI|DOCS|GENERAL|IOS|TVOS|WINDOWS)\]\s*?\[(BREAKING|FEATURE|ENHANCEMENT|BUGFIX)\]\s*?\[(.*)\]\s*?\-\s*?(.*)/ig;
+const includesReleaseNotes = danger.github.pr.body.toLowerCase().includes('release notes');
+const correctlyFormattedReleaseNotes = releaseNotesRegex.test(danger.github.pr.body);
+const releaseNotesCaptureGroups = releaseNotesRegex.exec(danger.github.pr.body);
+
+if (!includesReleaseNotes) {
+  const title = ':clipboard: Release Notes';
+  const idea = 'This PR appears to be missing Release Notes.';
+  warn(`${title} - <i>${idea}</i>`);
```

Comment:
Maybe use message here and below, since a PR may not necessarily _require_ release notes.

Reply:
I think we will require them in the PR template, but opt out of adding them to the final notes via keyword as discussed below. That ok?